org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -3376,20 +3820,22 @@ class ProgressParams {
 	/**
 	 * The progress token provided by the client or server.
 	 */
-	Either<String, Number> token
+	@NonNull
+	Either<String, Integer> token
```

Comment:
Not sure about this. The LSP spec says it's a number which is 64bit. Integer would only be 32bit.

Reply:
`ProgressToken` type is defined as `integer | string` in LSP 3.16 -- https://microsoft.github.io/language-server-protocol/specifications/specification-3-16/#progress. `integer` type defined as "in the range of -2^31 to 2^31 - 1" -- https://microsoft.github.io/language-server-protocol/specifications/specification-3-16/#number

Reply:
I see. I was coming from https://microsoft.github.io/language-server-protocol/specification#progress. Sad.